bot/glossary_command.go

```diff
@@ -0,0 +1,148 @@
+package bot
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/quintilesims/slackbot/db"
+	"github.com/quintilesims/slackbot/models"
+	glob "github.com/ryanuber/go-glob"
+	"github.com/urfave/cli"
+)
+
+// NewGlossaryCommand returns a cli.Command that manages !glossary
+func NewGlossaryCommand(store db.Store, w io.Writer) cli.Command {
+	return cli.Command{
+		Name:  "!glossary",
+		Usage: "manage the glossary",
+		Subcommands: []cli.Command{
+			{
+				Name:      "define",
+				Usage:     "add or set an entry in the glossary",
```

Comment:
If that is true, usage should change to just add and we should have a set entry or something right.

Reply:
naw since `rm` is a thing they can use that. However, I wouldn't be opposed to adding a `--force` flag to `add` (however, I'd expect that changset be propagated for every command, which would be more work for you).